src/main/java/com/wavefront/opentracing/WavefrontTracer.java

```diff
@@ -251,6 +253,11 @@ void reportWavefrontGeneratedData(WavefrontSpan span) {
       put(OPERATION_NAME_TAG, span.getOperationName());
       put(COMPONENT_TAG_KEY, span.getComponentTagValue());
     }};
+    String application = span.getSingleValuedTagValue(APPLICATION_TAG_KEY);
+    String service = span.getSingleValuedTagValue(SERVICE_TAG_KEY);
```

Comment:
We also need to add the application and service to the point tags, and make sure they get reported with the expected tags to Wavefront. Basically on the P3 page you should see the derived metrics for services that were reported by changing the app/service on a per span basis.